README.md

```diff
@@ -1,2 +1,6 @@
 # raspberry-pi
-Source code for project Vrijtap meant to run on the Raspberry Pi 3
```

Comment:
Deze line is weg nu?

Reply:
Wat voor IT-er ben je, dit is geen link, skill issue

Reply:
Deze review accepteer ik niet, laat ook maar, we mergen wel niet

Reply:
Mijn superieure lijn in markdown is te goed voor deze vieze README
![Uploading 20231111_152548.jpg…]()